ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/sftp_outbound_transfer.py

```diff
@@ -302,6 +303,100 @@ def _handle_s3_event(s3_object_key: str):
             send_notification(topic=partner_topic, notification=success_notif)
 
 
+def poll_queue():
```

Comment:
@dondevun I think my understanding is a bit unclear. On a high level, when a Peering Partner uploads a file to their folder in the bfd eft bucket, the eft sftp outbound transfer lambda will run and upload the file via CMS EFT SFTP (happy path) but if it fails, it'll send some messages to the SQS. Your new additions so far is to process these messages?

Reply:
So, when we were using Lambda, the python script would be called automatically and passed the S3 event, which gives the script information about the files that were uploaded... Now that we are moving to hosting the script in an EC2 instance, we don't have that luxury... So, my changes allow the script to continuously poll an SQS queue (every five minutes). If there is an update, S3 will put an event on that queue for us to read. Then we continue processing the S3 event as we've already been doing.

Reply:
@dondevun Okay makes sense. Will there be any changes made through terraform included in this ticket or just this python script change?

Reply:
There will be terraform changes, but it is in a different ticket that @mjburling is working on.

Reply:
Kk so this PR is about done, you just said that you need to include some tests, right?

Reply:
Yep. Will update with the tests shortly.